README.md

```diff
@@ -109,18 +113,36 @@ html = '<p>The sum of ' + a + ' and ' + b + ' plus ' + c +
 ```
 
 Lines should be broken into logical groups if it improves readability, such as splitting each expression of a ternary operator onto its own line even if both will fit on a single line.
+If possible, use ternary conditions between parentheses to ensure the logic, for the reader.
+
+```js
+var baz = (firstCondition( foo ) && secondCondition( bar )) ?
+    qux( foo, bar ) :
+    foo;
+```
+
+Never use multiple ternary conditions in the same instruction.
 
 ```js
-var baz = firstCondition( foo ) && secondCondition( bar ) ?
+// bad
+var baz = (firstCondition( foo ) && secondCondition( bar )) ?
```

Comment:
ok, fixing it...but notice that it was already there :p 
it wasn't me who added those spaces.